content/docs/tasks/telemetry/querying-metrics/index.md

```diff
@@ -93,14 +93,13 @@ of Istio metrics.
 
 The configured Prometheus add-on scrapes three endpoints:
 
-1. *istio-mesh* (`istio-mixer.istio-system:42422`): all Mixer-generated mesh
+1. *istio-mesh* (`istio-telemetry.istio-system:42422`): all Mixer-generated mesh
    metrics.
-1. *mixer* (`istio-mixer.istio-system:9093`): all Mixer-specific metrics. Used
+1. *mixer* (`istio-telemetry.istio-system:9093`): all Mixer-specific metrics. Used
    to monitor Mixer itself.
-1. *envoy* (`istio-mixer.istio-system:9102`): raw stats generated by Envoy (and
-   translated from Statsd to Prometheus).
+1. *envoy* (`istio-proxy:15090`): raw stats generated by Envoy.
```

Comment:
while we are here, can we also add entries for pilot, galley, and istio-policy  (and update the "The configured Prometheus add-on scrapes three endpoints:" to read "The configured Prometheus add-on scrapes the following endpoints:") ?

Reply:
added.